Restructure and improve docs (#84)
projectsyn/lieutenant-operator: docs/modules/ROOT/pages/index.adoc

= Project SYN: Lieutenant Operator

Welcome to the documentation of the Lieutenant Operator - a Project Syn tool.

image::syn.png[]

Keeping inventory about tenants and clusters. It's the "database" for the link:https://github.com/projectsyn/lieutenant-api[lieutenant-api]. It primarily holds information about the tenants and clusters.

> It also handles the management of some requirements like git repositories and secret management. It can automatically populate git repositories with skeleton files when a new cluster is added. It will generate link:https://github.com/projectsyn/steward[Steward] secrets for cluster bootstrapping.

Reviewer: ```suggestion
It also handles the management of some requirements like Git repositories and secret management: It can automatically populate Git repositories with template files when a new cluster is added. It will also generate a token to be used by https://github.com/projectsyn/steward[Steward].
```
